test/pointing_party_web/controllers/page_controller_test.exs

```diff
@@ -4,9 +4,11 @@ defmodule PointingPartyWeb.PageControllerTest do
   @username "test_user"
 
   describe "authenticated user" do
-    setup  %{conn: conn} do
-      conn = conn
-      |> Plug.Conn.assign(:username, @username)
+    setup %{conn: conn} do
+      conn =
+        conn
+        |> Plug.Conn.assign(:username, @username)
```

Comment:
I generally try to avoid using a pipe when there's only one function. (Chrisopher Adams's style guide also [recommends](https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines) avoiding the pipe operator when there's just one `|>`) What do you think about the following code?
```ex
conn = Plug.Conn.assign(conn, :username, @username)
```

I have a similar suggestion for [this code](https://github.com/elixirschool/pointing-party/pull/8/files#diff-51344aeb6586e385c77c3b2642a91c49R26).